.github/workflows/ci.yaml

```diff
@@ -0,0 +1,62 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - release
+  pull_request:
```

Comment:
If I understand [the GitHub Actions syntax](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions) properly, this means we won't run CI on branches that don't have PRs any more. Is that right? I think there was some discussion on Slack about this, but it can be helpful to test branches before opening a PR to catch issues earlier on. Can we change this section to just `on: [push, pull_request]` instead?

Reply:
Done: https://github.com/nextstrain/augur/pull/796/commits/3794d75f14e4ddccd948ce7d3b7bbdee741ab32e